Integrations/integration-Panorama.yml

```diff
@@ -2299,6 +2324,396 @@ script:
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('External Dynamic List:', edl_output, ['Name', 'Type', 'URL', 'Recurring', 'CertificateProfile', 'Description'], None, True),
+            'EntryContext': {
+                  "Panorama.EDLs(val.Name == obj.Name)": edl_output
+            }
+        })
+
+
+    @logger
+    def panorama_create_edl(edl_name, url, type_, recurring, certificate_profile = None, description = None):
+        params = {
+            'action': 'set',
+            'type': 'config',
+            'xpath': XPATH_OBJECTS + "external-list/entry[@name='" + edl_name + "']/type/" + type_,
+            'key': API_KEY
+        }
+        params['element'] = add_argument(url, 'url', False) + '<recurring><' + recurring + '/></recurring>' + add_argument(certificate_profile, 'certificate-profile', False) + add_argument(description, 'description', False)
+        result = http_request(
+            URL,
+            'POST',
+            params = params,
+        )
+        return result
+
+
+    def panorama_create_edl_command():
+        ''' Create an edl object '''
+        edl_name = demisto.args().get('name')
+        url = demisto.args().get('url')
+        type_ = demisto.args().get('type')
+        recurring = demisto.args().get('recurring')
+        certificate_profile = demisto.args().get('certificate_profile')
+        description = demisto.args().get('description')
+
+        edl = panorama_create_edl(edl_name, url, type_, recurring, certificate_profile, description)
+
+        edl_output = {
+            'Name': edl_name,
+            'URL': url,
+            'Type': type_,
+            'Recurring': recurring
+        }
+        if description:
+            edl_output['Description'] = description
+        if certificate_profile:
+            edl_output['CertificateProfile'] = certificate_profile
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': edl,
+            'ReadableContentsFormat': formats['text'],
+            'HumanReadable': 'EXternal Dynamic List was added successfully',
+            'EntryContext': {
+                  "Panorama.EDLs(val.Name == obj.Name)": edl_output
+            }
+        })
+
+
+    @logger
+    def panorama_edit_edl(edl_name, element_to_change, element_value):
+
+        edl_prev = panorama_get_edl(edl_name)
+        if '@dirtyId' in edl_prev:
+            return_error('Please commit the instance prior to editing the External Dynamic List')
+        edl_type = ''.join(edl_prev['type'].keys())
+        edl_output = {'Name': edl_name}
+        params = {
+            'action': 'edit',
+            'type': 'config',
+            'key': API_KEY,
+        }
+
+        if element_to_change == 'url':
+            params['xpath'] = XPATH_OBJECTS + "external-list/entry[@name='" + edl_name + "']/type/" + edl_type + "/" + element_to_change
```

Comment:
if you use the same code in all the lines(2506,2512,2518...) Think it is more readable to create it and put it in a ve before the switch case